core/src/main/java/com/sequenceiq/cloudbreak/structuredevent/rest/urlparsers/V4ExistingResourceRestUrlParser.java

```diff
@@ -17,7 +17,7 @@
 
     // Irregular requests with resource ID instead of resource name: v4/{workspaceId}/audits/{auditId}
     // Irregular GET requests with event but no resource name: v4/{workspaceId}/audits/zip and remaining patterns
-    private static final Pattern ANTI_PATTERN = Pattern.compile("v4/\\d+/(audits/.*|cluster_definitions/recommendation|image_catalogs/images"
+    private static final Pattern ANTI_PATTERN = Pattern.compile("v4/\\d+/(audits/.*|blueprints/recommendation|image_catalogs/images"
```

Comment:
Not a big deal but could you please also update the respective comment in `V4EventRestUrlParser`?

Reply:
yep